lib/Site.js

```diff
@@ -28,7 +28,6 @@ const SITE_CONFIG_DEFAULT = {
   ],
   ignore: [
     '_site/*',
-    '*.json',
```

Comment:
It is still possible for users to ignore `.json` files and the search will no have data for the autocomplete. Maybe we can indicate in the documentation or make sure `site.json` cannot be ignored even if the users specifies for it?

Reply:
Hmm. Authors may not want site.json to be accessible.

Reply:
The site.json is required to be accessible to provide the search results. Perhaps we can mention in the documentation that it is a requirement to have site.json to be accessible in order for search to work?

Reply:
If the file doesn't contain confidential data such as usernames/passwords, it should be fine to expose it.